### PR TITLE
feat: add TUN device documentation

### PR DESCRIPTION
**Pull request by deansheather**

### CODE REVIEW

File: admin/workspace-management/tun-device.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: TUN device
+description: Learn how to enable TUN devices for VPN usage within Coder.
+state: alpha
+---
+
+By default Coder workspaces don't contain a TUN device for security reasons,
```
Comment (jawnsy) [inline admin/workspace-management/tun-device.md:7]:
```suggestion
By default, Coder workspaces don't contain a TUN device for security reasons,
```

Is this actually true, security reasons? I don't think it should be a security issue, and writing this without context about the possible security risks may scare people away from using it

File: admin/workspace-management/tun-device.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+which means running a VPN in a Coder workspace is difficult. Coder offers an
+admin configuration setting that will automatically create a TUN device within
+all Kubernetes CVM workspaces.
+
+> At this time, Coder does not support other workspace types such as EC2 or
```
Comment (jawnsy) [inline admin/workspace-management/tun-device.md:12]: does not support _TUN devices for_ other workspace types

File: admin/workspace-management/tun-device.md (hunk 1 above)
Comment (deansheather) [reply admin/workspace-management/tun-device.md:7]: True, I guess I should remove the security bit

**Review by khorne3 — APPROVED** [review]

**Comment by deansheather** [comment]
@khorne3 Did you mean to add the two top commits to this PR?

- chore: document hostPaths mounted by CVMs
- chore:update shutdown behavior info

**Comment by khorne3** [comment]
@deansheather no, I didn't -- I'll merge on the release branch and it should be fixed

**Comment by deansheather** [comment]
@khorne3 feel free to merge this once you're ready in that case 👍

**Pull request merged**